force_wfmanager/central_pane/pareto_front.py

```diff
@@ -0,0 +1,82 @@
+from traits.api import (HasStrictTraits, List, Str, Tuple, Instance, Enum,
+                        Property, on_trait_change)
+
+from traitsui.api import View, UItem, Item, VGroup, HGroup
+
+from enable.api import Component, ComponentEditor
+
+from chaco.api import ArrayPlotData, Plot
+
+
+class ParetoFront(HasStrictTraits):
+    #: List of parameter names
+    value_names = List(Str)
```

Comment:
put this one and the evaluation steps in the common model and then pass the model as a reference. You are getting copies here.